### DISCUSSION TITLE
HowTo fix Google Chrome not starting after renaming the host in Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#890] [Comments]
# HowTo fix Google Chrome not starting after renaming the host in Ubuntu 24.04

My recent days were marked by the biennial (as in every two years) prematurely (I&rsquo;ll explain that in a later post) installation of a new Ubuntu LTS (long term supp…

https://kollitsch.dev/blog/2024/howto-fix-google-chrome-not-starting-after-renaming-the-host-in-ubuntu-2404/

**Comment by brycewray** [comment]
Hmm. Had a similar Chrome profile issue earlier this week when I upgraded my Linux-using old iMac from Fedora 39 to Fedora 40, but the hostname hadn’t changed. Thus, am not sure how comparable your situation is to mine, but it was an interesting article, nonetheless.

**Comment by davidsneighbour** [comment]
It could be an IP issue. If your IP address changed (I for instance use a static IP address, but after new installations it's assigned via router) that could lead to the same issue.

**Comment by davidsneighbour** [comment]
I think the main reason is not that the hostname changed but that Chrome somehow thought the profile was hijacked to a different computer altogether and then "locked" it pre-emptively.

**Reply by brycewray** [reply]
Would assume you’ve heard of the following by now:

https://linuxiac.com/do-not-try-to-upgrade-to-ubuntu-24-04-at-this-time/

> Ubuntu 24.04 LTS (Noble Numbat) was released just a few days ago. So you might want to jump to it from 22.04 LTS or 23.10 (Mantic Minotaur)—the excitement is big! But let me be direct and a bit intense here: JUST DON’T! I’ll explain why below.

**Reply by davidsneighbour** [reply]
Hehe, yes, and I remember thinking the last time to wait until they have the first "stable" release out. OCD won out one last time. I'll write up a summary in a bit. I am still working on some finer details that seem to be unable to cope with progress :D

**Comment by StephenDuMont** [comment]
I signed out in the google website, uninstalled google-chome using apt, rebooted and then opened chrome, seemed to work.

**Comment by marcellowmr** [comment]
It worked for me! Thank you!

**Comment by walter-sbn** [comment]
Good, this has worked. Had the issue after host renamed and Chrome updated.

**Comment by SpikeyMikie** [comment]
this was absolutely spot on, thanks so much for the easy fix!

**Comment by marmotas** [comment]
It worked for me, thanks!

**Comment by LeonHascal** [comment]
Thank you so much for this post. I never would have connected this problem with my having changed the hostname yesterday. I was just about to go nuclear on my system when I found this.  I simply removed all three Singleton files and it fixed everything.  👍
